classes/module/Module.php

```diff
@@ -2894,6 +2916,8 @@ public function adminDisplayWarning($msg)
      * add a info message to display at the top of the admin page.
      *
      * @param string $msg
+     *
+     * @return false|void
```

Comment:
I wonder if the actual statement wouldn't be:

```suggestion
     * @return false|null
```

Reply:
Prefer void where no return statement:
https://docs.phpdoc.org/guides/types.html#keywords
> void
> This is not the value that you are looking for. The tag associated with this type does not intentionally return anything. Anything returned by the associated element is incidental and not to be relied on.

Reply:
Well, using void would make sens if the method never returns anything (and I would remove the `@return` statement in that case). But if we need to list the possible values returned, `null` will be one of them, as this is how PHP behaves.

Reply:
Php is weird sometimes :thinking: